plugins/node.yaml

```diff
@@ -0,0 +1,163 @@
+    type: string
+    default: "*"
+    relevant_if:
+      source:
+        equals: kubernetes
+  - name: file_log_path
+    label: File Path
+    description: Specify a single path or multiple paths to read one or many files. You may also use a wildcard (*) to read multiple files within a directory.
+    type: strings
+    relevant_if:
+      source:
+        equals: file
+  - name: exclude_file_log_path
+    label: Exclude File Path
+    description: Specify a single path or multiple paths to exclude one or many files from being read. You may also use a wildcard (*) to exclude multiple files from being read within a directory.
+    type: strings
+    default: []
+    relevant_if:
+      source:
+        equals: file
+  - name: encoding
+    label: Encoding
+    description: Specify the encoding of the file(s) being read. In most cases, you can leave the default option selected.
+    type: enum
+    valid_values:
+      - utf-8
+      - utf-16le
+      - utf-16be
+    default: utf-8
+    relevant_if:
+      source:
+        equals: file
+  - name: start_at
+    label: Start At
+    description: Start reading file from 'beginning' or 'end'
+    type: enum
+    valid_values:
+      - beginning
+      - end
+    default: end
+# Set Defaults
+# {{$source := default "file" .source}}
+# {{$cluster_name := default "" .cluster_name}}
+# {{$pod_name := default "" .pod_name}}
+# {{$container_name := default "*" .container_name}}
+# {{$encoding := default "utf-8" .encoding}}
+# {{$start_at := default "end" .start_at}}
+
+# Pipeline Template
+pipeline:
+# {{ if eq $source "kubernetes" }}
+  - type: kubernetes_container
+    pod_name: '{{ $pod_name }}'
+    container_name: '{{ $container_name }}'
+    cluster_name: '{{ $cluster_name }}'
+    start_at: '{{ $start_at }}'
+    enable_nested_json_parser: false
+    exclude:
+      - '/var/log/containers/stanza*'
+# {{ end }}
+
+# {{ if eq $source "file" }}
+  - type: file_input
+    start_at: '{{ $start_at }}'
+    include:
+# {{ range $i, $fp := .file_log_path  }}
+      - '{{ $fp }}'
+# {{ end }}
+# {{ if .exclude_file_log_path }}
+    exclude:
+  # {{ range $i, $efp := .exclude_file_log_path  }}
+      - '{{ $efp }}'
+  # {{ end }}
+# {{ end }}
+# {{ if $encoding }}
+    encoding: '{{ $encoding }}'
+# {{ end }}
+    include_file_name: true
+    labels:
+      plugin_id: {{ .id }}
+# {{ end }}
+
+  - id: add_log_type
+    type: add
+    field: '$labels.log_type'
+    value: 'node'
+
+  - type: json_parser
+
+  - id: flatten_meta_req
+    type: flatten
+    if: '$record.meta.req != nil'
+    field: meta.req
+
+  - id: flatten_meta_res
+    type: flatten
+    if: '$record.meta.res != nil'
+    field: meta.res
+
+  - id: flatten_meta
+    type: flatten
+    if: '$record.meta != nil'
+    field: meta
+
+  - type: add
+    if: '$record.httpVersion != nill'
+    field: "protocol"
+    value: "http"
+
+  - type: move
+    if: '$record.httpVersion != nill'
+    from: httpVersion
+    to: protocol_version
```

Comment:
I suspect this is working, however, the example you gave in your pr does not have `protocol_version`.

Reply:
You are correct that is working. I had removed it from my last entry to confirm I didn't get errors if it wasn't present. Then I used that as the example.